airdrop/config.py

```diff
@@ -24,6 +24,8 @@
 SIGNER_PRIVATE_KEY = 'AIRDROP_SIGNER_PRIVATE_KEY'
 SIGNER_PRIVATE_KEY_STORAGE_REGION = ''
 
+NUNET_TOKEN_ADDRESS = ''
```

Comment:
Why is this blank if used

Reply:
This file get replaced from S3 config when code build triggers